dagmc_stats/DagmcStats.py

```diff
@@ -40,3 +39,24 @@ def __init__(self, filename, populate=False):
         self._vol_data = pd.DataFrame(
             columns=['vol_eh', 'surf_per_vol', 'coarseness'])
         self._vol_data = self._vol_data.set_index('vol_eh')
+        
+        self.entity_types = [types.MBVERTEX, types.MBTRI, types.MBENTITYSET]
```

Comment:
I think this still makes more sense as a class variable, and it can be accessed directly in the member function, as described below.  That said, you may have discussed a reason that it should not be a class variable...?

Reply:
I don't understand what you are asking for. Is this not a class variable being set in the init?